blog_post.md

```diff
@@ -147,31 +147,32 @@ where the last equality is a general result for geometric series. This means tha
+### Experiments and hyperparameters
+Since divergence can now be quantified, we use it as a metric to compare which algorithms exhibit more divergence than others. **We say an algorithm exhibits more divergence if the fraction of runs in which soft divergence occurs is higher.** We refer to Memory Replay and Target Networks as DQN's "tricks". The improvement that each of the tricks brings to DQN is measured against the **baseline** model, DQN without tricks, or *vanilla* DQN.
+*We thus compare 4 different setups for each environment*: without tricks (vanilla agent), with Memory Replay (memory agent), with target networks (target agent), and with both tricks (DQN / memory+target agent).
+
+We run each experiment with **random seeds from 1 to 25** to achieve more statistically sound results, while taking into account our computational budget. If the maximal absolute Q-value, predicted in any of the last 20 training episodes, is above the threshold $$\frac{1}{1-\gamma}$$, we say soft divergence occurs. 
+<!--- At the end, we compare the configurations by counting how many times each of them has diverged. -->
+
+All the agents are **trained for 700 episodes** which we found to be enough for them to learn to win the games. For better exploration, we use an **$$\epsilon$$-greedy** strategy which is **linearly annealed from 1 to 0.1 during the first 400 episodes**, and keep it fixed after. The discount factor is **$$\gamma = 0.99$$** for all the environments.
 
-All the experiments are run for 700 episodes which has been found to be enough for the agents to learn to win the games. For better exploration, we use an $$\epsilon$$-greedy strategy, which we linearly anneal from 1 to a minimum of 0.1 during the first 400 episodes, and keep it fixed after. The discount factor is fixed to 0.99 for all the environments.
+Another hyperparameter is the **frequency of target network updates** (whenever the technique is used) and we empirically find **400, 2000, 2000** to work well for **Mountain Car, Cart Pole and Acrobot respectively** (environments are described below). No extensive hyperparameter search has been executed since the focus of our work is not state-of-the-art performance but to compare the importance of the methods instead. The values of **the parameters are selected manually for the configuration with no tricks** and kept fixed for all other configurations of the respective environment.
```

Comment:
```suggestion
Another hyperparameter is the **frequency of target network updates** (whenever the technique is used) and we empirically find **400, 2000, 2000** to work well for **Mountain Car, Cart Pole and Acrobot respectively**. No extensive hyperparameter search has been executed since the focus of our work is not state-of-the-art performance but to compare the importance of the methods instead. The values of **the parameters are selected manually for the configuration with no tricks** and kept fixed for all other configurations of the respective environment.
```

Reply:
Remove the "environments are described below"